modules/@apostrophecms/doc/index.js

```diff
@@ -347,6 +347,15 @@ module.exports = {
           submitted: 1,
           aposLocale: 1
         });
+        await self.db.createIndex({
+          type: 1,
+          aposDocId: 1,
+          aposLocale: 1
+        });
+        await self.db.createIndex({
+          aposDocId: 1,
```

Comment:
Still helpful to be able to do this fast when the query is not locked to one type.